sdk/digitaltwins/azure-digitaltwins/azure/digitaltwins/_digitaltwins_client.py

```diff
@@ -0,0 +1,547 @@
+
+    def get_event_route(self, event_route_id, **kwargs):
+        # type: (str, **Any) -> "models.EventRoute"
+        """Get an event route.
+        Status codes:
+        200 (OK): Success.
+        404 (Not Found): There is no event route with the provided id.
+
+        :param event_route_id: The Id of the event route.
+        :type event_route_id: str
+        :returns: The EventRoute object.
+        :rtype: object
+        :raises :class: `~azure.core.exceptions.HttpResponseError`
+        """
+        return self._client.event_routes.get_by_id(event_route_id, **kwargs)
+
+    def list_event_routes(self, max_item_count=-1, **kwargs):
+        # type: (int, **Any) -> "models.EventRoute"
+        """Retrieves all event routes.
+        Status codes:
+        200 (OK): Success.
+        400 (Bad Request): The request is invalid.
+
+        :param max_item_count: The maximum number of items to retrieve per request.
+            The server may choose to return less than the requested max.
+        :type max_item_count: str
+        :returns: An iterator like instance of the EventRouteCollection.
+        :rtype: ~azure.core.paging.ItemPaged[~azure.digitaltwins.models.EventRouteCollection]
+        :raises :class: `~azure.core.exceptions.HttpResponseError`
+        """
+        digital_twin_models_list_options = {'max_item_count': -1}
+        if max_item_count != -1:
+            digital_twin_models_list_options= {'max_item_count': max_item_count}
+
+        return self._client.event_routes.list(digital_twin_models_list_options, **kwargs)
+
+    def upsert_event_route(self, event_route_id, event_route, **kwargs):
+        # type: (str, "models.EventRoute", **Any) -> object
+        """Create or update an event route.
+        Status codes:
+        200 (OK): Success.
+        400 (Bad Request): The request is invalid.
+
+        :param event_route_id: The Id of the event route to create or update.
+        :type event_route_id: str
+        :param event_route: The event route data.
+        :type event_route: ~azure.digitaltwins.models.EventRoute
+        :returns: None
+        :rtype: None
+        :raises :class: `~azure.core.exceptions.HttpResponseError`
+        """
+        return self._client.event_routes.add(event_route_id, event_route, **kwargs)
+
+    def delete_event_route(self, event_route_id, **kwargs):
+        # type: (str, **Any) -> object
+        """Delete an event route.
+        Status codes:
+        200 (OK): Success.
+        404 (Not Found): There is no event route with the provided id.
+
+        :param event_route_id: The Id of the event route to delete.
+        :type event_route_id: str
+        :returns: None
+        :rtype: None
+        :raises :class: `~azure.core.exceptions.HttpResponseError`
+        """
+        return self._client.event_routes.delete(event_route_id, **kwargs)
+
+    def query_twins(self, query_specification, **kwargs):
```

Comment:
I belive this should be a paged/iterable operation - is that's how it's done in other languages?

Reply:
Fixed